README.md

```diff
@@ -1 +1,4 @@
-Todo :)
\ No newline at end of file
+# Kubetel
+
```

Comment:
Not sure if you sould remove the todo yet :P

Reply:
I just wanted to fix the typos in the code first still working on the actual readme